subtests/docker_cli/systemd/systemd.py

```diff
@@ -0,0 +1,81 @@
+"""
+Summary
+-------
+Testing basic container as systemd service
+
+Operational Summary
+---------------------
+This is to test container as systemd service using a predefined unit file
+(p4321.service) and a script p4321-server.py. An image is built with this
```

Comment:
The docstrings are parsed as RST, so it's good to add double-back-quotes around anything that should be in a courier/system font.  Makes them easier to read.  e.g.

This is to test container as systemd service using a predefined unit file
(``p4321.service``) and a ``script p4321-server.py``. An image is built with this
script added. Systemd starts a container using this image & the container
writes current time to port ``4321``. From host, the socket is read and the value
is checked to see if it is a digit. Finally, container is stopped through
systemd and system is cleaned up.